app/imports/components/comments/Comments.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+
+import { Comment } from '/imports/components/comments/Comment.jsx';
+
+export const Comments = ({comments, commentsLimit, postId}) => {
```

Comment:
So children of the reactive component can be functional all like this? Because of comments are reactive but only with data that changes higher in the component stack I guess